packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -439,9 +472,22 @@ export function signInActor({ services }: SignInMachineOptions) {
         },
         async forceNewPassword(context, event) {
           const { user, formValues } = context;
-          const { password } = formValues;
+          let {
+            password,
+            confirm_password,
+            phone_number,
+            country_code,
+            ...rest
+          } = formValues;
+
+          let phoneNumberWithCountryCode;
+          if (phone_number) {
+            phoneNumberWithCountryCode =
+              `${country_code}${phone_number}`.replace(/[^A-Z0-9+]/gi, '');
+            rest = { ...rest, phone_number: phoneNumberWithCountryCode };
```

Comment:
Doesn't have to be on this PR, but should we use @slaymance's `parsePhoneNumber` here from #826?